drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationDocksHandler.java

```diff
@@ -78,12 +77,12 @@
                                       "PLAY_CIRCLE",
                                       new DockPlaceHolderPlace(TEST_RUNNER_REPORTING_PANEL),
                                       perspectiveIdentifier);
-        result.add(reportDock.withSize(450).withLabel(DefaultWorkbenchConstants.INSTANCE.TestReport()));
+        result.add(reportDock.withSize(450).withLabel("TEST REPORT"/*DefaultWorkbenchConstants.INSTANCE.TestReport()*/));
```

Comment:
Is this change required? Why use an hard coded string?

Reply:
@yesamer 
No reason for hardcoded string, you're right

Reply:
Done